graphql_test.go

```diff
@@ -351,6 +351,88 @@ func TestBasic(t *testing.T) {
 	})
 }
 
+type testEmbeddedStructResolver struct{}
+
+func (_ *testEmbeddedStructResolver) Course() courseResolver {
+	return courseResolver{
+		CourseMeta: CourseMeta{
+			Name:       "Biology",
+			Timestamps: Timestamps{CreatedAt: "yesterday", UpdatedAt: "today"},
+		},
+		Instructor: Instructor{Name: "Socrates"},
+	}
+}
+
+type courseResolver struct {
+	CourseMeta
+	Instructor Instructor
+}
+
+type CourseMeta struct {
+	Name string
+	Timestamps
+}
+
+type Instructor struct {
+	Name string
+}
+
+type Timestamps struct {
+	CreatedAt string
+	UpdatedAt string
+}
```

Comment:
Same here. Can you, please, move the definition of these structs inside of the test `TestEmbeddedStruct`?